.github/workflows/remark.yml

```diff
@@ -0,0 +1,23 @@
+name: Remark
+
+on:
+  push:
+    paths:
+    - '**.md'
+  pull_request:
+    paths:
+    - '**.md'
+
+jobs:
+  remark:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.0.0
+    - name: Setup Node.js
+      uses: actions/setup-node@v1.1.0
+    - name: Install  remark
+      run: npm install remark-cli remark-lint remark-lint-maximum-line-length remark-preset-lint-recommended
+    - name: Check *.md files
+      run: find -name "*.md" ! -path "./node_modules/*" -exec ./node_modules/.bin/remark {} -u lint -f > /dev/null \;
```

Comment:
```suggestion
      shell: bash
      run: git ls-files -z '*.md' | xargs -0 -n 1 -I {} ./node_modules/.bin/remark {} -u lint -f > /dev/null \;
```

Reply:
shell isn't needed here, since it runs on ubuntu anyway.

Out of interest: Why is `git ls-files` + `xargs` better than find?

Reply:
`find` does searching on all directories including `.git`. In contrast, `git ls-files` just uses index files.

Reply:
Thanks, good to know!